src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -92,47 +96,21 @@ export function uiRenderMixin(osdServer, server, config) {
       );
       const uiSettings = osdServer.newPlatform.start.core.uiSettings.asScopedToClient(soClient);
 
-      const darkMode =
+      const configDarkMode =
         !authEnabled || request.auth.isAuthenticated
```

Comment:
i feel like this logic isn't hooked into from the security plugin

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5831

Reply:
Yeah, that's a good idea, although all I changed here was the naming.